bin/appliance_console

```diff
@@ -67,7 +67,8 @@ include ManageIQ::ApplianceConsole::Prompts
 RESTORE_LOCAL   = "Local file".freeze
 RESTORE_NFS     = "Network File System (NFS)".freeze
 RESTORE_SMB     = "Samba (SMB)".freeze
-RESTORE_OPTIONS = [RESTORE_LOCAL, RESTORE_NFS, RESTORE_SMB, ManageIQ::ApplianceConsole::CANCEL].freeze
+RESTORE_S3      = "AWS S3 (S3)".freeze
+RESTORE_OPTIONS = [RESTORE_LOCAL, RESTORE_NFS, RESTORE_SMB, RESTORE_S3, ManageIQ::ApplianceConsole::CANCEL].freeze
```

Comment:
I don't see this being used in this file ... Are we sure these need to be here anymore?

cc @NickLaMuro

Reply:
Looks like no, but I didn't want to remove them as part of this PR, and felt that I should be consistent.  In hindsight probably just should have figured it out and removed them.